src/sentry/static/sentry/app/views/organizationGroupDetails/eventToolbar.tsx

```diff
@@ -75,29 +75,59 @@ class GroupEventToolbar extends React.Component<Props> {
     const {event, organization, location} = this.props;
 
     return (
-      <QuickTraceWrapper>
-        <ErrorBoundary mini>
-          <QuickTraceQuery event={event} location={location} orgSlug={organization.slug}>
-            {results =>
-              results.isLoading ? (
-                <Placeholder height="24px" />
-              ) : results.error || results.trace === null ? (
-                '\u2014'
+      <ErrorBoundary mini>
+        <QuickTraceQuery event={event} location={location} orgSlug={organization.slug}>
+          {results => (
+            <React.Fragment>
+              {results.isLoading || results.error || results.trace === null ? (
+                ''
               ) : (
```

Comment:
You don't need to insert empty strings when there is not trace. Instead you can conditionally render the link with something like this.

```suggestion
              {!results.isLoading && results.error === null && results.trace !== null && (
```